docs/quick-start/intellij-setup.md

```diff
@@ -1,45 +1,72 @@
+## Prerequisites
+Running the code in IntelliJ requires the following prerequisites:
 
 - IntelliJ idea installed on your local machine
 - Java 11
 - Maven 3
-- The codebase cloned into a IntelliJ project and imported as a maven project with a Java 11 SDK configured.
+- The codebase cloned into an IntelliJ project and imported as a maven project with a Java 11 SDK configured.
 
-## Setting up the maven module
+## Setting the maven project
 Go to **File > Project structure** or <kbd>&#8984;</kbd> + <kbd>;</kbd> after doing this you should should see the following menu.
 ![Project structure](https://i.imgur.com/3JjeqR5.png){.center}
-At this point you should set the project SDK to Java 11 as well as the language level. 
-From here we go on to the modules tab, press the blue + button and choose the **new module** and the maven option.
-After completing this step you should get the following menu,
+At this point, you should set the project SDK to Java 11 and the language level. 
+We go on to the modules tab, press the blue + button and choose the **new module** and the maven option.
+After completing this step, you should get the following menu,
 ![New module](https://i.imgur.com/1Kr5Nkr.png){.center}
 You should click the **next** button.
-At this point you will be prompted to add choose a name and location of the module.
-The location should be the **app** directory in the datapreparer root and the name can be what ever you choose. The completed configration should look like this,
-![Complete module configuration](https://i.imgur.com/CC6ujkZ.png)
-After this you should press finish. You have now set up a maven module for the application.
-> *IMPORTANT*: you will need to copy and paste the pom.xml into from the repo app folder,
->  it is overwritten locally by the IntelliJ.
-> It is avaliable [here]().
+You will see a prompt to choose the name and location of the module at this point.
+Keep the name it gives you. 
+The completed configuration should look like this,
+![Complete module configuration](https://i.imgur.com/VJj89Oh.png){.center}
+From here, you should get a maven project sidebar.
+Your IntelliJ should look something like this,
+![IntelliJ with maven config](https://i.imgur.com/IvhVgix.png){.center}
+
+## Fixing Java version in maven
+You now need to set the Java version in the pom.xml file.
+You can do this by adding the following lines to pom.xml inside the app directory:
+
+```xml
+<plugin>
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-compiler-plugin</artifactId>
+    <version>3.8.1</version>
+    <configuration>
+        <source>11</source>
+        <target>11</target>
+    </configuration>
+</plugin>
+```
+
+## Adding the datapreparer maven configuration
+We now need to add the maven configuration for the *app* directory, which contains the application source code.
+If you look at the maven tab, you will see a small plus button.
+This screenshot shows the tab and the plus button,
```

Comment:
Instead of these line you could simplify say "Click on the  small plus button in the maven tab, as shown: "

Reply:
Done